src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +288,26 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                if (
+                    features.has(
```

Comment:
```suggestion
                    not ctx["commit_id"]
                    and features.has(
```

https://github.com/getsentry/sentry/blob/94847f884003a1b2571f0aa6969e715a089449ab/src/sentry/api/endpoints/project_stacktrace_link.py#L59